resdk/resources/collection.py

```diff
@@ -64,6 +68,16 @@ def data(self, value):
         self._data = value
         self._data_hydrated = False
 
+    def add_data(self, *data):
```

Comment:
Nice. I like that you have supported this. Write in docs what can be the input parameter. We could/should? also support adding or removing data by slugs. This would probably be the most common use case when working in the interactive Python shell.